content/hi/version_control.md

```diff
@@ -0,0 +1,17 @@
+---
+title: वर्शन कण्ट्रोल (Version Control)
+status: Completed
+category: तकनीकी
+---
+
+## यह क्या है
+
+सोर्स कण्ट्रोल (या वर्शन कण्ट्रोल) किसी डोकुमेंट में परिवर्तनों को ट्रैक करने और प्रबंधित करने का अभ्यास है। यह एक ऐसी सिस्टम है जो समय के साथ फ़ाइल या फ़ाइलों के सेट में परिवर्तनों को रिकॉर्ड करती है ताकि आप बाद में स्पेसिफिक वर्शनों को याद कर सकें।
+
+## समस्या
+
+वर्शन कण्ट्रोल सिस्टम्स निम्नलिखित समस्याओं को हल करने के लिए काम करती है, जैसे की समय के साथ डोकुमेंट या कोडबेस का बैक अप लेना, एकाधिक उपयोगकर्ताओं को ओवरलैपिंग परिवर्तन होने पर टकराव को हल करने की इजाजत देता है, और समय के साथ परिवर्तनों का हिस्ट्री रिकॉर्ड करता है। एप्लिकेशन कोड अक्सर प्रमुख व्यावसायिक प्रक्रियाओं के लिए जटिल और महत्वपूर्ण हो सकता है, इसलिए यह ट्रैक करना इम्पोर्टेन्ट है कि किसने क्या, कब और क्यों बदला। इसके अलावा, कई, यदि अधिकांश एप्लिकेशन नहीं हैं, तो कई डेवलपर्स द्वारा संशोधित किए जाते हैं, और विभिन्न डेवलपर्स द्वारा शुरू किए गए परिवर्तनों के बीच अक्सर टकराव होते हैं।
+
+## समाधान
+
+वर्शन कण्ट्रोल डेवलपर्स को तेजी से आगे बढ़ने और एफिशिएंसी को बनाए रखने में मदद करता है, जबकि परिवर्तनों का रिकॉर्ड स्टोर करता है और टकराव को हल करने की सुविधा प्रदान करता है। यह उन्हें एप्लिकेशन कोड को एक रिपॉजिटरी में स्टोर करने और सहयोग को सरल बनाने की अनुमति देता है। मॉडर्न एप्लीकेशन डेवलपमेंट अपने कोड को स्टोर करने के लिए git जैसे वर्शन कण्ट्रोल सिस्टम पर बहुत अधिक निर्भर करता है।
```

Comment:
टकराव to भेद

Reply:
But @Garima-Negi was suggested to use टकराव

Reply:
In that case we can opt it out